web/src/overlays.js

```diff
@@ -2,14 +2,14 @@ import $ from "jquery";
 import Micromodal from "micromodal";
 
 import * as blueslip from "./blueslip";
-import * as browser_history from "./browser_history";
 
 let $active_overlay;
 let close_handler;
 let open_overlay_name;
 
 const pre_open_hooks = [];
 const pre_close_hooks = [];
+const exit_overlay_hook = [];
```

Comment:
This should be named `close_settings_hooks`.

Reply:
Made the suggested changes, Changes are ready for review now.